apps/Directory.mk

```diff
@@ -64,7 +64,7 @@ all: nothing
 
 define SDIR_template
 $(1)_$(2):
-	$(Q) $(MAKE) -C $(1) $(2) TOPDIR="$(TOPDIR)" APPDIR="$(APPDIR)"
+	$(Q) $(MAKE) -C $(1) $(2) TOPDIR="$(TOPDIR)" APPDIR="$(APPDIR)" EXTDIR="$(EXTDIR)"
```

Comment:
it should do so via this change:
https://github.com/Samsung/TizenRT/pull/278/files#diff-194321126c6a80da39301224fafb4840